driver/web/driver.go

```diff
@@ -43,51 +50,50 @@ type Config struct {
 	// by DNS round robin, like a kubernetes headless service.
 	DNSDiscovery bool `json:"dns_discovery" yaml:"dns_discovery"`
 
-	// Connections to the specified CDP server are proxied by replicant
-	Proxied bool `json:"proxied" yaml:"proxied"`
-}
+	// Path to chrome binary
+	BinaryPath string `json:"binary_path" yaml:"binary_path"`
 
-// New creates a new web driver
-func New(c Config) (d *Driver) {
-	return &Driver{c}
-}
+	// Arguments for launching chrome
+	BinaryArgs []string `json:"binary_path" yaml:"binary_path"`
 
-// Type returns this driver type
-func (d *Driver) Type() (t string) {
-	return "web"
-}
+	// Interval for recycling chrome processes
+	RecycleInterval time.Duration
 
-// New creates a web transaction
-func (d *Driver) New(config transaction.Config) (tx transaction.Transaction, err error) {
-	txn := &Transaction{}
-	txn.proxied = d.config.Proxied
-
-	serverURL := d.config.ServerURL
-	s, ok := config.Inputs["cdp_address"]
-	if ok {
-		serverURL, ok = s.(string)
-		if !ok {
-			return nil, fmt.Errorf("driver/web: unexpected value for cdp_address: %#v", s)
-		}
-	}
+	// for testing only
+	testing bool
```

Comment:
Is this supposed to exist?

Reply:
yes, this allows testing with a local context only for html elements and is currently the only way for having tests for this driver. (tests are under way)